### PR TITLE
Fix color scheme

### PR DESCRIPTION
**Pull request by felipesere**
👋 I like [x]it so I noticed that this plugin didn't fully work.
The fix was to extend the tree-sitter schema (I'll do a different PR for that)
and to match the names of the highlights.

There might be some more smaller cleanups coming 😅

### CODE REVIEW

**Comment by felipesere** [comment]
Fixes #5

File: lua/xit/init.lua [hunk 1]
```diff
@@ -302,9 +294,9 @@ M.setup = function(opts)
   local parser_config = require('nvim-treesitter.parsers').get_parser_configs()
   parser_config.xit = {
     install_info = {
-      url = 'https://github.com/synaptiko/tree-sitter-xit',
+      url = 'https://github.com/felipesere/tree-sitter-xit',
       files = { 'src/parser.c' },
-      revision = '0.1',
+      revision = 'master',
```
Comment (synaptiko) [inline lua/xit/init.lua]: Would you consider to create PR for `tree-sitter-xit` as well and then we could revert this and I would release `0.2` with your changes?

**Review by synaptiko — CHANGES_REQUESTED** [review]
Thank you for the contribution. Sorry, I totally missed these issues and PRs and only noticed now. But your changes looks great, I'd love to accept it. Please see my comment below.

**Comment by felipesere** [comment]
Woah, totally forgot about this and just circled back to setup a nice todo list again.

Here is the PR to update tree-sitter before we bump this and merge: https://github.com/synaptiko/tree-sitter-xit/pull/2

**Comment by synaptiko** [comment]
@felipesere Thank you, PR in `tree-sitter-xit` approved.

**Comment by felipesere** [comment]
Merge it and I can update this.

**Comment by synaptiko** [comment]
@felipesere Done, there is [0.2](https://github.com/synaptiko/tree-sitter-xit/releases/tag/0.2) version with your changes.

**Comment by felipesere** [comment]
Done

File: lua/xit/init.lua [hunk 2]
```diff
@@ -89,50 +95,33 @@ local set_mappings = function(M, augroup, options)
...
-      vim.keymap.set('n', '<leader>x', M.delete_task, { buffer = true, silent = true })
-      vim.keymap.set('n', '<leader>fo', function()
-        M.filter_tasks({ 'open_task', 'ongoing_task' })
-      end, { buffer = true, silent = true })
-      vim.keymap.set('n', '<leader>fc', function()
-        M.filter_tasks({ 'checked_task' })
-      end, { buffer = true, silent = true })
-      vim.keymap.set('i', '<CR>', M.create_new_task_in_insert_mode, { buffer = true, silent = true })
-      vim.keymap.set('i', '<S-CR>', M.create_indented_line_in_insert_mode, { buffer = true, silent = true })
+      for key_strokes, interaction in pairs(options.keymaps or {}) do
```
Comment (synaptiko) [inline lua/xit/init.lua:119]: This means the plugin lost its default keybindings, right?

It would be good to add example (maybe with the default keybindings) into README and document the "breaking" change.

Would you be willing to handle it in another PR?

**Review by synaptiko — APPROVED** [review]
Thank you 👍
